src/server/command_registry.h

```diff
@@ -33,6 +33,8 @@ enum CommandOpt : uint32_t {
   NOSCRIPT = 0x100,
   BLOCKING = 0x200,  // implies REVERSE_MAPPING
   GLOBAL_TRANS = 0x1000,
+
+  CUSTOM_JOURNAL = 1U << 10,
```

Comment:
I'll change the ones above to bitshift + or, because it hard to understand it in this way